YAKL_atomics.h

```diff
@@ -195,6 +195,40 @@
     __device__ __forceinline__ void atomicMax(unsigned long long int &update , unsigned long long int value) {
       ::atomicMax( &update , value );
     }
+  #endif
+
+  #ifdef __USE_OPENMP45__
+    template <class T> inline void atomicAdd(T &update, T value) {
+      #pragma omp atomic update 
+        update += value;
+    }
+    template <class T> inline void atomicMin(T&update, T value) {
+      #pragma omp critical
+      {
+        if (value < update){update = value;}
+
+      }
+      //T tmp;
+      //#pragma omp atomic read
+      //  tmp = update;
+      //if (tmp > value) {
+      //  #pragma omp atomic write
+      //    update = value;
+      //}
+    }
+    template <class T> inline void atomicMax(T &update, T value) {
+      #pragma omp critical
+      {
+        if(value > update){update = value;}
```

Comment:
Please change to `update = value > update ? value : update`

Reply:
Update but still fails atomic test.